sample/F8/sessions

```diff
@@ -0,0 +1,1470 @@
+   {
+      "short_abstract":"",
+      "id":20,
+      "subtitle":"a-startup-journey-from-an-emerging-market",
+      "comments":"",
+      "language":"",
+      "slides":"",
+      "audio":"",
+      "video":"",
+      "signup_url":"",
+      "state":"accepted",
+      "session_type":{
+         "id":4,
+         "name":"20 minutes session",
+         "length":"00.20"
+      },
+      "track":{
+         "id":10,
+         "name":"Workplace",
+         "color":"#BF532E"
+      },
+      "title":"A Startup Journey from an Emerging Market",
+      "long_abstract":"Gary Urteaga shares his entrepreneurial journey from his initial curiosity about Silicon Valley, various web & app projects, milestones as a startup founder, and his work helping entrepreneurs transform big ideas into startups that solve the humanity's grand challenges.",
+      "microlocation":{
+         "id":7,
+         "name":"Hall B"
+      },
+      "start_time":"2017-04-18T14:00:00-07:00",
+      "end_time":"2017-04-18T14:20:00-07:00",
+      "speakers":[
+         {
+            "city":"",
+            "heard_from":"",
+            "icon":"",
+            "long_biography":"",
+            "small":"",
+            "photo":"",
+            "thumbnail":"",
+            "short_biography":"",
+            "speaking_experience":"",
+            "sponsorship_required":"",
+            "organisation":"Facebook",
+            "id":30,
+            "name":"Gary Urteag"
+         }
+      ]
+   },
+   {
+      "short_abstract":"",
+      "id":21,
+      "subtitle":"building-high-quality-javascript-tools",
+      "comments":"",
+      "language":"",
+      "slides":"",
+      "audio":"",
+      "video":"",
+      "signup_url":"",
+      "state":"accepted",
+      "session_type":{
+         "id":6,
+         "name":"50 minutes session",
+         "length":"00.50"
+      },
+      "track":{
+         "id":3,
+         "name":"Engineering and Open source",
+         "color":"#E8782C"
+      },
+      "title":"Building High-Quality JavaScript Tools",
+      "long_abstract":"Yarn and Jest are popular and widely adopted open source projects. In this session, we’ll present our roadmap for these projects and share our experiences – good and bad – in releasing and maintaining popular open source projects.",
+      "end_time":"2017-04-18T15:20:00-07:00",
+      "microlocation":{
+         "id":6,
+         "name":"Hall F"
+      },
+      "start_time":"2017-04-18T14:30:00-07:00",
+      "speakers":[
+         {
+            "city":"",
+            "heard_from":"",
+            "icon":"",
+            "long_biography":"",
+            "small":"",
+            "photo":"",
+            "thumbnail":"",
+            "short_biography":"",
+            "speaking_experience":"",
+            "sponsorship_required":"",
+            "organisation":"Facebook",
+            "id":31,
+            "name":"Christoph Pojer "
+         },
+         {
+            "organisation":"Facebook",
+            "id":32,
```

Comment:
please use same speaker info. one has more attributes than the other. please use the same format for each speaker and find images for speakers on the net.